kazoo/tests/conftest.py

```diff
@@ -4,8 +4,10 @@
 
 
 def pytest_exception_interact(node, call, report):
-    if hasattr(node._testcase, "cluster"):
```

Comment:
Why does this disappear? Because now we're always trying to get ZK logs... are there tests that don't have the "cluster" attr? If so, we're needlessly trying/failing to fetch the ZK logs IIUC...

Reply:
Without this change I was not able to get the ZK logs, for some reasons.

Examples:
- test that failed recently without this change, no "Zookeeper cluster logs": https://github.com/python-zk/kazoo/actions/runs/4661094003/jobs/8249964080
- test that failed with this change, you get some "Zookeeper cluster logs": https://github.com/python-zk/kazoo/actions/runs/4586252822/jobs/8098978225

Maybe I am missing something obvious

Reply:
What you've changed it to sounds reasonable to me... I was hoping the `"cluster"` attr would be auto-set, but looks like it's probably missing in some cases, and we don't want to have to remember to always manually set it.